src/Client.cpp

```diff
@@ -183,6 +210,9 @@ void Client::OnAfterCreated(CefRefPtr<CefBrowser> browser)
 
     CEF_REQUIRE_UI_THREAD();
 
+    m_requestHandler->m_messageRouterBrowserSide = CefMessageRouterBrowserSide::Create(constants::messageRouterConfig);
```

Comment:
Oh cool, I will fix this to be concurrent. Any clues on how to do this would be appreciated but I'm sure I can work it out if not.